cuckoo/utils.py

```diff
@@ -69,3 +69,174 @@ def grouper(
         tuple(entry for entry in iterable if entry is not None)
         for iterable in zip_longest(*iterables, fillvalue=None)
     )
+
+
+class Prop:
```

Comment:
I have decided to use an `_` _at the end_ of the method name. reasoning:
- `and`, `or`, `with` are reserved words and cannot be used as method names
- `_` at the beginning of the method indicates a private method
- I have added the `_` to all methods to be consistent

Let me know if you have ideas for better naming. thanks

Reply:
Hmm, interested to try this out. Personally, I think typing the raw dictionary is fine, but I can see some benefit here, with things like `Prop("size_dc_mw").asc_()`.

Reply:
Yeah I am unlikely to use this right away, but maybe once I see it in action it'll be convinced